.github/workflows/stale.yml

```diff
@@ -0,0 +1,26 @@
+name: "Stale"
+
+# This uses Github actions minutes. To learn more: https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions
+
+on:
+  schedule:
+    - cron: "0 0 1 * *"
```

Comment:
Should this be `0 1 * * *` if it's scheduled to run once a day at 1 am?

Reply:
Yes you're right, the trailing `1` should've been a `*`